vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -295,6 +295,12 @@ void vtkF3DImguiActor::RenderDropZone()
   if (this->DropZoneVisible)
   {
     ImGuiViewport* viewport = ImGui::GetMainViewport();
+
+    if (viewport->WorkSize.x < 50 || viewport->WorkSize.y < 50)
```

Comment:
I think the guard should be much smaller, like 10x10. I know that the text is already not fitting at that size but I can image someone willing to see the dropzone with smaller window.